libbeat/kibana/testdata/5.x-libbeat.json

```diff
@@ -0,0 +1,6 @@
+{
+  "fieldFormatMap": "{\"group_enabled.message\":{\"id\":\"url\",\"params\":{\"inputFormat\":\"string\"}}}",
```

Comment:
I suggest instead of keeping two files `libbeat.json` and `libbeat.old.json` under `libbeat/_meta/kibana/5.x/index-pattern/`, to have the two files under the `kibana/testdata` directory together with a `fields.yml` file.  For a better test, I suggest to use the JSON file and the `fields.yml` from a Beat, instead of libbeat, maybe Metricbeat as it's the most complex one.